control-plane/config/rbac/peeringacceptor_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit peeringacceptors.
```

Comment:
Everything in config/rbac was also generated but doesn't exist for other CRs, I think we can remove?